### PR TITLE
Update contributors.yml

### PR DESCRIPTION
**Pull request by doomspork**

### CODE REVIEW

File: _data/contributors.yml [hunk 1]
```diff
@@ -22,17 +25,35 @@
     - Just the beginning
     - Configuration Demystified
     - Ecto query composition
+    - 'Umbrellas: only when it rains?'
```
Comment (doomspork) [inline _data/contributors.yml]: It's not liking this for some reason, going to work with @burden to sort it out.

Comment (notactuallypagemcconnell) [reply _data/contributors.yml]: Might be the quotes

Comment (doomspork) [reply _data/contributors.yml]: It doesn't like it without them either 😞

File: _data/contributors.yml [hunk 2]
```diff
@@ -22,17 +25,35 @@
...
+  contributions:
+    - translator
+    - blog
+  languages:
+    - gr
+  articles:
+    - Reviewing Elixir in Action, Second Edition
+    - Reviewing Functional Web Development with Elixir, OTP, and Phoenix
+
+- github: notactuallypagemcconnell
```
Comment (doomspork) [inline _data/contributors.yml]: @notactuallypagemcconnell I update some of your info.  If you want to submit a PR with your intro and others that'd be an easy hacktoberfest contribution 😁

**Pull request merged**